test/tests/api/v2_0/skupack_tests.py

```diff
@@ -0,0 +1,376 @@
+            json.dump(self.__config_json, f)
+        f.close()
+
+        self.__tasks = {
+            "friendlyName": "Flash Quanta BMC",
+            "injectableName": "Task.Linux.Flash.unique.Bmc",
+            "implementsTask": "Task.Base.Linux.Commands",
+            "options": {
+              "file": None,
+              "downloadDir": "/opt/downloads",
+              "commands": [
+                "sudo /opt/socflash/socflash_x64 -b /opt/uploads/bmc-backup.bin",
+                "sudo curl -T /opt/uploads/bmc-backup.bin {{ api.files }}/{{ task.nodeId }}-bmc-backup.bin",
+                "sudo /opt/socflash/socflash_x64 -s option=x flashtype=2 if={{ options.downloadDir }}/{{ options.file }}"
+              ]
+            },
+            "properties": {
+              "flash": {
+                "type": "bmc",
+                "vendor": {
+                  "quanta": {}
+                }
+              }
+            }
+        }
+        with open('/tmp/tarball/tasks/tasks.json', 'w') as f:
+            json.dump(self.__tasks, f)
+        f.close()
+
+        self.__workflows = {
+            "friendlyName": "noop-sku-graph",
+            "injectableName": "Graph.noop-example",
+            "tasks": [
+                {
+                    "label": "noop-1",
+                    "taskName": "Task.noop"
+                },
+                {
+                    "label": "noop-2",
+                    "taskName": "Task.noop",
+                    "waitOn" : {
+                        "noop-1" :"finished"
+                    }
+                }
+            ]
+        }
+        with open('/tmp/tarball/workflows/workflows.json', 'w') as f:
+            json.dump(self.__workflows, f)
+        f.close()
+
+        self.__template = {
+            "friendlyName": "Flash Quanta BMC",
+            "injectableName": "Task.Linux.Flash.unique.Bmc",
+            "implementsTask": "Task.Base.Linux.Commands",
+            "options": {
+                "file": None,
+                "downloadDir": "/opt/downloads",
+                "commands": [
+                    "sudo /opt/socflash/socflash_x64 -b /opt/uploads/bmc-backup.bin",
+                    "sudo curl -T /opt/uploads/bmc-backup.bin {{ api.files }}/{{ task.nodeId }}-bmc-backup.bin",
+                    "sudo /opt/socflash/socflash_x64 -s option=x flashtype=2 if={{ options.downloadDir }}/{{ options.file }}"
+                ]
+            },
+            "properties": {
+                "flash": {
+                    "type": "bmc",
+                    "vendor": {
+                        "quanta": {}
+                    }
+                }
+            }
+        }
+        with open('/tmp/tarball/templates/template.json', 'w') as f:
+            json.dump(self.__template, f)
+        f.close()
+
+        os.chdir("/tmp/tarball")
+        subprocess.call(["tar", "czf", "/tmp/tarball/mytest.tar.gz", "config.json", "profiles", "static", "tasks", "templates", "workflows"])
```

Comment:
Suggestion, tarfile library is pretty useful for generating tarballs natively: 
https://docs.python.org/2/library/tarfile.html